eclair-node/src/main/scala/fr/acinq/eclair/api/serde/JsonSerializers.scala

```diff
@@ -135,10 +135,10 @@ class PrivateKeySerializer extends CustomSerializer[PrivateKey](_ => ( {
   case _: PrivateKey => JString("XXX")
 }))
 
-class ChannelVersionSerializer extends CustomSerializer[ChannelVersion](_ => ( {
+class ChannelConfigSerializer extends CustomSerializer[ChannelConfigOptions](_ => ( {
   null
 }, {
-  case x: ChannelVersion => JString(x.bits.toBin)
+  case x: ChannelConfigOptions => JString(x.bytes.toBin)
```

Comment:
A resolution to feature names (like we do in `FeaturesSerializer`) would be nice. Something like:

```
{
  "channelConfig": [
    "FundingPubKeyBasedChannelKeyPath"
  ],
  "channelType": "AnchorOutputs"
}
```

Reply:
I completely agree for the `channelConfig`, I'll add names.
For the `channelType`, it already displays the list of human-readable features (`"option_anchor_outputs": "optional") so we should be good.

Reply:
Done in https://github.com/ACINQ/eclair/pull/1852/commits/aba4fa59782c83993358a87899344f1f7b3c324b